backend/main.py

```
@@ -0,0 +1,95 @@
+import random
+from colorama import Fore
```

Ruff (0.8.2)

`colorama.Fore` imported but unused

Remove unused import: `colorama.Fore`

(F401)